actions/poll_status.yaml

```diff
@@ -3,7 +3,7 @@ name: poll_status
 runner_type: run-python
 description: Polls an URL until the web service says that the long process has finished
 enabled: true
-entry_point: lib/poll_status.py
+entry_point: poll_status.py
```

Comment:
It makes it more difficult to make the testing works, since you have to add each folder to the `PYTHONPATH` manually. Moving them into the same folder made this easier.